packages/phoenix/src/utils.rs

```diff
@@ -33,6 +33,11 @@ macro_rules! validate_bps {
     }
 }
 
+/// Validate if the current timestamp is within desired timestamp
+pub fn is_tx_active(env: &Env, deadline: u64) -> bool {
+    env.ledger().timestamp() < deadline
+}
```

Comment:
```suggestion
pub fn is_before_deadline(env: &Env, deadline: u64) -> bool {
    env.ledger().timestamp() <= deadline
}
```

Also I wonder if there is a point of creating a function for a one line if statement.

Reply:
will get rid of the helper, doesn't do much